var/spack/repos/builtin/packages/hsa-rocr-dev/package.py

```diff
@@ -24,8 +24,10 @@ class HsaRocrDev(CMakePackage):
     libraries = ["libhsa-runtime64"]
 
     version("master", branch="master")
+    version("5.6.0", sha256="30875d440df9d8481ffb24d87755eae20a0efc1114849a72619ea954f1e9206c")
     version("5.5.1", sha256="53d84ad5ba5086ed4ad67ad892c52c0e4eba8ddfa85c2dd341bf825f4d5fe4ee")
     version("5.5.0", sha256="8dbc776b56f93ddaa2ca38bf3b88299b8091de7c1b3f2e481064896cf6808e6c")
+    version("5.4.4", sha256="6a0c6300ccf812b6341097e999394389a6343d8fcc92740a8636f61631e2d276")
```

Comment:
Is it safe to assume that this version does not also need to be added to the `hip` and other packages here that don't have it?